Add "link to documentation" button

ollimeier opened the pull request:
Fixes #142 

This is how it looks like (including a tooltip):
<img width="732" alt="Screenshot 2024-12-06 at 06 01 08" src="https://github.com/user-attachments/assets/a2cf4ad0-310b-48f6-a0a2-ac2e2580190b">

justvanrossum commented on FontraPakMain.py:
```diff
@@ -149,24 +149,33 @@ def __init__(self, port):
-        layout = QVBoxLayout()
+        layout = (
+            QGridLayout()
+        )  # Helpful: https://www.pythontutorial.net/pyqt/pyqt-qgridlayout/
```
Please move that comment to before the `layout = ...` statement, so that black doesn't wrap it in parentheses.

ollimeier replied:
done.

justvanrossum commented on FontraPakMain.py:
```diff
@@ -149,24 +149,33 @@ def __init__(self, port):
+        buttonDocs = QPushButton("?", self)
+        buttonDocs.setToolTip("Link to <b>documentation</b>")
+        buttonDocs.setSizePolicy(QSizePolicy.Policy.Fixed, QSizePolicy.Policy.Fixed)
+        buttonDocs.clicked.connect(lambda: webbrowser.open("http://docs.fontra.xyz"))
```
```suggestion
        buttonDocs.clicked.connect(lambda: webbrowser.open("https://docs.fontra.xyz"))
```

justvanrossum commented on FontraPakMain.py:
```diff
@@ -149,24 +149,33 @@ def __init__(self, port):
-        layout.addWidget(button)
-        layout.addWidget(self.label)
+        buttonDocs = QPushButton("?", self)
+        buttonDocs.setToolTip("Link to <b>documentation</b>")
```
```suggestion
        buttonDocs.setToolTip("Link to documentation")
```

ollimeier replied:
done. see https://github.com/googlefonts/fontra-pak/pull/143/commits/2c73fb0212aaa5e6cf47dab9dcbbceeabadb9e92

justvanrossum reviewed (commented):
I left some minor comments/suggestions, otherwise looks good, thanks!

justvanrossum commented:
I changed my mind about the tooltip, I think it should read:

    Open documention website

ollimeier commented:
> I changed my mind about the tooltip, I think it should read:
> 
> ```
> Open documention website
> ```

I think you meant `documentation` instead of `documention`. I changed it. done.
